docs/UserGuide.adoc

```diff
@@ -59,13 +59,15 @@ items with `...` after them can have multiple instances. Order of parameters are
 Put a `p` before the phone / email / address prefixes to mark it as `private`. `private` details can only
 be seen using the `viewall` command.
 
+
+
```

Comment:
Stray line break? It is a good practice to check the diff of the PR and fix any problems before submitting it.